docs/src/forc/workspaces.md

```diff
@@ -0,0 +1,29 @@
+# Workspaces
+
+A *workspace* is a collection of one or more packages, namely *workspace members*, that are managed together.
+
+The key points for workspaces are:
+
+* Common `forc` commands available for a single package can also be used for a workspace, like `forc build` or `forc deploy`.
+* All packages share a common [`Forc.lock`] file which resides in the root directory of the workspace.
+
+For workspaces, `Forc.toml` accepts following fields:
```

Comment:
Workspace manifests are declared within `Forc.toml` files and support the following fields: